doc/03-tracking-events.md

```diff
@@ -4,6 +4,54 @@ Snowplow has been built to enable you to track a wide range of events that occur
 
 We provide several built-in event classes to help you track different kinds of events. When instantiated, their objects can be passed to the `Snowplow.track()` method to send events to the Snowplow collector. The event classes range from single purpose ones, such as `ScreenView`, to the more complex but flexible `SelfDescribing`, which can be used to track any kind of user behaviour. We strongly recommend using `SelfDescribing` for your tracking, as it allows you to design custom event types to match your business requirements. [This post](https://snowplowanalytics.com/blog/2020/01/24/re-thinking-the-structure-of-event-data/) on our blog, "Re-thinking the structure of event data" might be informative here.
 
+## Auto-tracked events
```

Comment:
Good idea to move this up front! Just thought that we could make the heading more specific since there is only one kind of auto-tracked events:

```suggestion
## Auto-tracked view events
```